engines/agi/words.cpp

```diff
@@ -123,6 +123,31 @@ int Words::loadDictionary(const char *fname) {
 	return errOK;
 }
 
+int Words::loadExtendedDictionary(const char *sierraFname) {
+	Common::String fnameStr = Common::String(sierraFname) + ".extended";
+	const char *fname = fnameStr.c_str();
+
+	Common::File fp;
+
+	if (!fp.open(fname)) {
+		warning("loadWords: can't open %s", fname);
+		return errOK; // err_BadFileOpen
```

Comment:
?

Reply:
It's based on `loadDictionary` which has the exact following code:

	if (!fp.open(fname)) {
		warning("loadWords: can't open %s", fname);
		return errOK; // err_BadFileOpen
	}